database/misc/mongodb_parameter.py

```diff
@@ -0,0 +1,244 @@
+Sponsored by E.T.A.I. http://www.etai.fr/
+
+This file is part of Ansible
+
+Ansible is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Ansible is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+DOCUMENTATION = '''
+---
+module: mongodb_parameter
+short_description: Change an administrative parameter on a MongoDB server.
+description:
+    - Change an administrative parameter on a MongoDB server.
+version_added: "2.1"
+options:
+    login_user:
+        description:
+            - The username used to authenticate with
+        required: false
+        default: null
+    login_password:
+        description:
+            - The password used to authenticate with
+        required: false
+        default: null
+    login_host:
+        description:
+            - The host running the database
+        required: false
+        default: localhost
+    login_port:
+        description:
+            - The port to connect to
+        required: false
+        default: 27017
+    login_database:
+        description:
+            - The database where login credentials are stored
+        required: false
+        default: null
+    replica_set:
+        description:
+            - Replica set to connect to (automatically connects to primary for writes)
+        required: false
+        default: null
+    database:
+        description:
+            - The name of the database to add/remove the user from
+        required: true
+    ssl:
+        description:
+            - Whether to use an SSL connection when connecting to the database
+        default: False
+    param:
+        description:
+            - MongoDB administrative parameter to modify
+        required: true
+        default: null
+    value:
+        description:
+            - MongoDB administrative parameter value to set
+        required: true
+        default: null
+    param_type:
+        description:
+            - Define the parameter value (str, int)
+        required: false
+        default: str
+
+notes:
+    - Requires the pymongo Python package on the remote host, version 2.4.2+. This
+      can be installed using pip or the OS package manager. @see http://api.mongodb.org/python/current/installation.html
+requirements: [ "pymongo" ]
+author: "Loic Blot (@nerzhul)"
+'''
+
+EXAMPLES = '''
+# Set MongoDB syncdelay to 60 (this is an int)
+- mongodb_parameter: param="syncdelay" value=60 param_type="int"
+'''
+
+RETURN = '''
+changed:
```

Comment:
removed `changed` here, it is a common return. Only document the returns the module provides please.